src/reference/community-plugins.md

```diff
@@ -295,6 +295,7 @@ generate dependency lockfiles and check for changes at build time.
 
 - [wartremover](https://github.com/wartremover/wartremover): flexible Scala
   linting tool. <!-- 728 stars -->
+- [sbt-findsecbugs](https://github.com/code-star/sbt-findsecbugs): Run Spotbugs with FindSecBugs plugin from sbt.
```

Comment:
develop branch is currently for sbt 2.x documentation.

sbt 1.x plugins are listed in https://github.com/sbt/website/blob/1.x/src/reference/01-General-Info/02-Community-Plugins.md, and it's ordered by GitHub stars within each section.